gradoop-temporal/src/test/java/org/gradoop/temporal/model/impl/TemporalGraphTest.java

```diff
@@ -193,4 +198,25 @@ public void testFromLogicalGraph() throws Exception {
     loadedVertices.forEach(this::checkDefaultTemporalElement);
     loadedEdges.forEach(this::checkDefaultTemporalElement);
   }
+
+  /**
+   * Test the
+   * {@link TemporalGraph#fromLogicalGraph(LogicalGraph, TimeIntervalExtractor, TimeIntervalExtractor, TimeIntervalExtractor)}  } method.
+   */
+  @Test
+  public void testFromLogicalGraphWithTimeIntervalExtractors() throws Exception {
+
+    String path = TemporalGraphTest.class.getResource("/datacsv/").getFile();
```

Comment:
I like this approach, I would just give the file a better name, maybe put them under `/data/csv/socialnetwork`